lib/publicConfig.ts

```diff
@@ -6,8 +6,8 @@ if (!network) throw "Missing NEXT_PUBLIC_NETWORK"
 const testNetUrl = process.env.NEXT_PUBLIC_TEST_NET_URL
 if (!testNetUrl) throw "Missing NEXT_PUBLIC_TEST_NET_URL"
 
-const crescendoNetUrl = process.env.NEXT_PUBLIC_CRESCENDO_URL
-if (!crescendoNetUrl) throw "Missing NEXT_PUBLIC_CRESCENDO_URL"
+const previewnetUrl = process.env.NEXT_PUBLIC_PREVIEWNET_URL
```

Comment:
using `PREVIEW_NET` elsewhere

Reply:
Does it matter? Just left it same format as the other was with testnet